site/src/pages/UserSettingsPage/SSHKeysPage/SSHKeysPage.tsx

```diff
@@ -33,27 +35,39 @@ export const SSHKeysPage: React.FC = () => {
   return (
     <>
       <Section title={Language.title} description={Language.description}>
-        {!sshKey && (
+        {authState.matches("signedIn.ssh.gettingSSHKey") && (
           <Box p={4}>
             <CircularProgress size={26} />
           </Box>
         )}
```

Comment:
We'll never have the progress spinner AND the code below, right? Let's use if/else here to make that clear.